docs/integrations-in-rancher/cluster-api/overview.md

```diff
@@ -0,0 +1,210 @@
+- Disable the `embedded-cluster-api` feature in Rancher.
+- Delete the `mutating-webhook-configuration` and `validating-webhook-configuration` webhooks that are no longer needed.
+:::
+
+#### Installation
+
+- From your browser, access Rancher Manager and explore the **local** cluster.
+- Using the left navigation panel, go to `Apps` -> `Repositories`.
+- Click `Create` to add a new repository.
+- Enter the following:
+    - **Name**: `turtles`.
+    - **Index URL**: https://rancher.github.io/turtles.
+- Wait for the `turtles` repository to have a status of `Active`.
+- Go to `Apps` -> `Charts`.
+- Filter for `turtles`.
+- Click `Rancher Turtles - the Cluster API Extension`
+- Click `Install` -> `Next` -> `Install`.
+
+This will use the default values for the Helm chart, which are good for most installations. If your configuration requires overriding some of these defaults, you can either specify the values during installation from the Rancher UI or, alternatively, you can opt for the [manual installation via Helm](./overview.md#via-helm-install). If you are interested in learning more about the available values, you can check the Rancher Turtles [Helm chart reference guide](https://docs.rancher-turtles.com/docs/reference-guides/rancher-turtles-chart/values).
+
+The installation may take a few minutes and once completed you will see the following new deployments in the cluster:
+- `rancher-turtles-system/rancher-turtles-controller-manager`
+- `rancher-turtles-system/rancher-turtles-cluster-api-operator`
+- `capi-system/capi-controller-manager`
+
+### Via Helm Install
+
+This section walks through different Helm installation options for Rancher Turtles depending on CAPI operator use:
+- [Install Rancher Turtles with Cluster API Operator as a dependency](#install-rancher-turtles-with-cluster-api-operator-as-a-helm-dependency).
+- [Install Rancher Turtles without Cluster API Operator](#install-rancher-turtles-without-cluster-api-operator-as-a-helm-dependency).
+
+The Cluster API Operator is required for installing Rancher Turtles and you can choose whether you want to take care of this dependency yourself or let the Rancher Turtles Helm chart manage it for you. We recommend [installing as a dependency](#install-rancher-turtles-with-cluster-api-operator-as-a-helm-dependency) for the sake of simplicity, but the best option may depend on your specific configuration.
+
+CAPI Operator allows for handling the lifecycle of Cluster API providers using a declarative approach, extending the capabilities of `clusterctl`. If you want to learn more about it, you can refer to [Cluster API Operator book](https://cluster-api-operator.sigs.k8s.io/).
+
+:::info prerequisite
+Before installing Rancher Turtles in your Rancher environment, Rancher's `embedded-cluster-api` functionality must be disabled. This includes also cleaning up Rancher-specific webhooks that otherwise would conflict with CAPI ones.
```

Comment:
```suggestion
Before installing Rancher Turtles in your Rancher environment, Rancher's `embedded-cluster-api` functionality must be disabled. This also includes cleaning up Rancher-specific webhooks that otherwise would conflict with CAPI ones.
```